README.md

```diff
@@ -29,6 +29,7 @@
     - [Android](#android)
     - [.NET](#net)
   - [Videos](#videos)
+  - [Blogs](#blogs)
```

Comment:
```suggestion
```

Reply:
Let's remove this and then we should be good 👍

Reply:
Done 👍🏼